common/app/renderers/DotcomRenderingService.scala

```diff
@@ -60,12 +61,20 @@ class DotcomRenderingService extends GuLogging with ResultWithPreconnectPreload
       endpoint: String,
       timeout: Duration = Configuration.rendering.timeout,
   )(implicit request: RequestHeader): Future[WSResponse] = {
+
+    val start = currentTimeMillis
+
     val resp = ws
       .url(endpoint)
       .withRequestTimeout(timeout)
       .addHttpHeaders("Content-Type" -> "application/json")
       .post(payload)
 
+    resp.foreach(_ => {
+      DCRMetrics.DCRLatencyMetric.recordDuration(currentTimeMillis - start)
```

Comment:
Just to verify my understanding here, every time `currentTimeMillis` is used it will return the latest time? I assume this is a Scala thing but I don't see a function being invoked (e.g. using `()`) so thought I'd double check

Reply:
Yep, its a scala thing! It does get invoked every time but I'm not a huge fan of this syntax because of this ambiguity, I've updated it!